app/features/quizzify/tools.py

```diff
@@ -239,8 +239,13 @@ def split_loaded_documents(self, loaded_documents: List[Document]) -> List[Docum
     def create_vectorstore(self, documents: List[Document]):
         if self.verbose:
             logger.info(f"Creating vectorstore from {len(documents)} documents")
-        
-        self.vectorstore = self.vectorstore_class.from_documents(documents, self.embedding_model)
+
+
+        logger.info("Before self.vectorstore_class.from_documents")
```

Comment:
While using logger.info here is fine for general logging, it might be helpful to include more specific details in the log message. The logger.info method is typically used to log informational messages that highlight the progress of the application at a coarse-grained level. This could be an opportunity to provide more context that could be useful for tracking the flow of execution or understanding the state of the application at this point.

For example:
logger.info(f"Starting vector store initialization using {self.vectorstore_class.__name__}. Preparing to load documents: {document_list}")